### PR TITLE
Copy changes, add min and max cost to calculator

### LINKED ISSUE #1361: Calculator text changes
Based on feedback from DEDP:

- [x] The price range should be $100 - 1000 (ideally this reange should come from the dashboard API
    See [FinancialAidCalculator.js#L144](https://github.com/mitodl/micromasters/blob/9e4c9a3be52b2f1ab5ca5bc71646f70662d2a44d/static/js/containers/FinancialAidCalculator.js#L144)
- [x] Change the line at [FinancialAidCalculator.js#L145](https://github.com/mitodl/micromasters/blob/9e4c9a3be52b2f1ab5ca5bc71646f70662d2a44d/static/js/containers/FinancialAidCalculator.js#L145) to be `based on your ability to pay`
- [x] Change the line at [FinancialAidCalculator.js#L63](https://github.com/mitodl/micromasters/blob/9e4c9a3be52b2f1ab5ca5bc71646f70662d2a44d/static/js/containers/FinancialAidCalculator.js#L63) to be `I am aware that I may be asked to verify the reported income with documentation.`

### PR DESCRIPTION
**Pull request by aliceriot**
#### What are the relevant tickets?

closes #1361 

#### What's this PR do?

This makes some copy changes to the financial aid calculator, and also adds in the max and min prices for the current program.

#### How should this be manually tested?

Make sure the copy looks right, and make sure that the same max and minimum prices that show up in the `FinancialAidCard` show up in the calculator.

### CODE REVIEW

File: static/js/containers/FinancialAidCalculator.js [hunk 1]
```diff
@@ -141,8 +150,8 @@ const FinancialAidCalculator = ({
     title="Cost Calculator"
   >
     <div className="copy">
-      { `The cost of courses in the ${title} Micromasters varies between $50 and $1000,
-      based on your income and ability to pay.`}
+      { `The cost of courses in the ${title} Micromasters varies between ${minPossibleCost} and ${maxPossibleCost},
```
Comment (noisecapella) [inline static/js/containers/FinancialAidCalculator.js]: They should still use `formatPrice` at least so they have a dollar sign

Comment (aliceriot) [inline static/js/containers/FinancialAidCalculator.js]: couldn't I just insert a dollar sign in the template?

**Comment by aliceriot** [comment]
@noisecapella from @roberthouse54, copy in both places should be changed to:

`The cost of courses in the ${program.title} varies between ${min} and ${max}, depending on your income and ability to pay.`

**Comment by aliceriot** [comment]
@noisecapella copy updated!

**Comment by noisecapella** [comment]
Looks good :+1:

**Pull request merged**